### ISSUE TITLE
Enable scroll wrap-around for list of points

### ISSUE THREAD

**Issue by HJarausch** [osmandapp/OsmAnd#9889]
Doing "Plan a route" for an existing route, one can display the list of waypoints. Currently it's hard to get at the end of this list, if this is very long. 
So, please do a wrap-around to the end of the list if one scrolls up at the very beginning of this list. 
Thanks for considering this tiny addition, 
Helmut

**Comment by thompehw** [comment]
I believe this request is about the difficulty of editing the end of a recorded GPX track of thousands of points, as recently discussed in [this thread](https://groups.google.com/forum/#!topic/osmand/S-N87cGyOkw) of the users' group.

Another way of  making the end of the track easily accessible would be to add a "Reverse Track" option, which would also provide this extra function.

I am another user who would appreciate this being addressed sometime - thanks!

**Comment by vshcherb** [comment]
Yes it's quite tricky. BTW you can select point directly on the map and do some actions with these points. Probably we can add action to scroll to it in the list
